.github/workflows/gnmi-tests.yml

```diff
@@ -0,0 +1,40 @@
+# This workflow will install Python dependencies, run tests
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: gnmi-pytest-tests
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [ '3.10', '3.9' ]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Checkout gnmi-tools
+        uses: actions/checkout@v3
+        with:
+           repository: 'ConfD-Developer/gnmi-tools'
+           path: 'gnmi-tools'
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v4
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install dependencies
+        run: |
+          pip install -r requirements.txt
+          pip install grpcio-tools
```

Comment:
grpcio-tools is already part of the requirements file, this line should not be needed.